examples/ratelimit/config/example.yaml

```diff
@@ -27,3 +27,13 @@ descriptors:
         rate_limit:
           unit: second
           requests_per_unit: 1
+      - key: bay
+        rate_limit:
+          unit: second
+          requests_per_unit: 5
```

Comment:
Does specifying unit/requests_per_unit make sense? Probably not? Also, can you document the feature in README.md?

Reply:
Thanks for looking at this @mattklein123. Removed `requests_per_unit`. The `unit` cannot be removed right now, it complains about it being of an invalid format. Do we wanna make it so that we can omit the `unit` too?

Also added an example to the README.